content/configuration/communication/_index.md

```diff
@@ -0,0 +1,108 @@
+  teams:
+    enabled: false
+    appID: 'APPLICATION_ID'
+    appPassword: 'APPLICATION_PASSWORD'
+    port: 3978
+    notiftype: short                    # Change notification type short/long you want to receive.
+                                        # notiftype is optional and default notification type is short (if not specified)
+
+  # Settings for Discord
+  discord:
+    enabled: false
+    token: 'DISCORD_TOKEN'	            # BotKube Bot Token
+    botid: 'DISCORD_BOT_ID'             # BotKube Application Client ID
+    channel: 'DISCORD_CHANNEL_ID'       # Discord Channel id for receiving BotKube alerts
+    notiftype: short                    # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
+
+  # Settings for ELS
+  elasticsearch:
+    enabled: false
+    awsSigning:
+      enabled: false                    # enable awsSigning using IAM for Elastisearch
+                                        # hosted on AWS, if true make sure AWS
+                                        # environment variables are set.
+                                        # Ref: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
+      awsRegion: 'us-east-1'            # AWS region where Elasticsearch is deployed
+      roleArn: ''                       # AWS IAM Role arn to assume for credentials,
+                                        # use this only if you dont want to
+                                        # use the EC2 instance role or not
+                                        # running on AWS instance
+    server: 'ELASTICSEARCH_ADDRESS'     # e.g https://example.com:9243
+    username: 'ELASTICSEARCH_USERNAME'  # Set creds if using Basic Auth
+    password: 'ELASTICSEARCH_PASSWORD'
+    # ELS index settings
+    index:
+      name: botkube
+      type: botkube-event
+      shards: 1
+      replicas: 0
+
+  # Settings for Webhook
+  webhook:
+    enabled: false
+    url: 'WEBHOOK_URL'                 # e.g https://example.com:80
+```
+
+The default configuration can be found at:
+- for Helm chart in [values.yaml](https://github.com/infracloudio/botkube/blob/master/helm/botkube/values.yaml),
+- and for all-in-one deployment spec in [deploy-all-in-one.yaml](https://github.com/infracloudio/botkube/blob/master/deploy-all-in-one.yaml).
+
+## Environment variables
+
+The individual communication settings can be specified via environment variables. It takes priority over the configuration specified in the file.
+
+To construct the environment variable name, take any property from the [configuration file](#file-syntax) and make it uppercase. Use the underscore for properties that are nested. That's all - you have the environment variable name!
+
+For example, such configuration property from YAML:
+```yaml
+communications:
+  slack:
+    token: 'SLACK_API_TOKEN'
+```
+is represented by `COMMUNICATIONS_SLACK_TOKEN` environment variable.

```

Comment:
Not sure about the word `represented` 🤔 
I would use something like `And this is the corresponding environment variable: `COMMUNICATIONS_SLACK_TOKEN`.